frontend/src/app/testResults/TestResultsList.tsx

```diff
@@ -65,7 +66,8 @@ function testResultRows(
   testResults: any,
   setPrintModalId: SetStateAction<any>,
   setMarkErrorId: SetStateAction<any>,
-  setDetailsModalId: SetStateAction<any>
+  setDetailsModalId: SetStateAction<any>,
+  setTextModalId: SetStateAction<any>
```

Comment:
I think this may be a copy/paste left over from my code ?

Reply:
hahaha plagiarism